config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml

```diff
@@ -727,6 +727,7 @@ periodics:
   annotations:
     testgrid-dashboards: sig-scalability-benchmarks
     testgrid-tab-name: scheduler-perf
+    testgrid-alert-email: patrick.ohly@intel.com, kubernetes-sig-scheduling@googlegroups.com
```

Comment:
This group is not the right audience...
Can you create a new group and include every approver? https://github.com/kubernetes/k8s.io/blob/main/groups/sig-scheduling/groups.yaml

Reply:
I'm not target to the approvers only, but the sig scheduling folks, make sense? I believe it's the community's responsibilities.

Reply:
@alculquicondor: any suggestion what I should call that group? "kubernetes-sig-scheduling-approvers"?

Reply:
just `sig-scheduling-approvers`, as the domain is `@kubernetes.io`

Reply:
alternatively, it can be `sig-scheduling-alerts`, where other people can ask to be added even if they are not approvers.

I recently updated `kueue-alerts` for this.

Reply:
Let's go with `sig-scheduling-alerts` because that's what the group is being used for. I can add myself to that group instead of adding myself just for this job.

Reply:
See https://github.com/kubernetes/k8s.io/pull/6876